docs/mllib-linear-methods.md

```diff
@@ -257,10 +257,10 @@ applying the logistic function
 \mathrm{f}(z) = \frac{1}{1 + e^{-z}}
 \]`
 where $z = \wv^T \x$.
-By default, if $\mathrm{f}(\wv^T x) > 0.5$, the outcome is positive, or
-negative otherwise, though unlike linear SVMs, the raw output of the logistic regression
-model, $\mathrm{f}(z)$, has a probabilistic interpretation (i.e., the probability
-that $\x$ is positive).
+By default, if $\mathrm{f}(\wv^T x) > 0.5$, the outcome is positive, else it is negative.
+Logistic regression is distinct from say linear SVMs in its formally being a Bayesian model, albeit trivial one: rather than producing directly an 'input-output machine', the conditional distribution of the output given the input is modeled explicitly through the function $\mathrm{f}$ above; this model can be and is then used to provide definite outputs for definite inputs.
```

Comment:
Even if this is accurate, I don't think it's an improvement. This drops the key points, that the output of logistic regression may be interpreted as a probability. I don't believe this description clarifies anything for Spark users.